src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -104,5 +129,54 @@ void shouldThrowAnExceptionWhenNameContainsIllegalLetters() throws Exception {
         result.andExpect(status().isBadRequest());
     }
 
+    @Test
+    @DisplayName("Should return empty list of wallets and http ok status")
+    void shouldReturnEmptyList() throws Exception {
+        // when
+        MockHttpServletResponse result = mockMvc.perform(get("/api/wallet")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().is2xxSuccessful())
+                .andReturn()
+                .getResponse();
+
+        // then
+        assertThat(result.getStatus()).isEqualTo(200);
```

Comment:
Nope. You got numeric value 200 instead of `HttpStatus.OK` or field constant

```java
import org.springframework.http.HttpStatus;

private static final HttpStatus HTTP_STATUS_OK = HttpStatus.OK;

(...)
        // then
        assertThat(result.getStatus()).isEqualTo(HTTP_STATUS_OK);
(...)
```
[HttpStatus - javadoc](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpStatus.html)

Reply:
- [ ] correct tests to assert http status instead of its numerical value

Reply:
```HttpStatus.OK``` is a enum

When we take a look at ```getStatus()``` method inside ```MockHttpServletResponse.class``` we see that it actually return int value:

```
    public int getStatus() {
        return this.status;
    }
```

So proposed assertion:
```assertThat(result.getStatus()).isEqualTo(HTTP_STATUS_OK);``` will always fail for 
`private static final HttpStatus HTTP_STATUS_OK = HttpStatus.OK;`

I have changed the code to below form:
```assertThat(result.getStatus()).isEqualTo(HttpStatus.OK.value());```